src/test/java/com/jcabi/ssh/mock/MkCommandTest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2014-2015, jcabi.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the jcabi.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.ssh.mock;
+
+import java.io.ByteArrayOutputStream;
+import org.apache.sshd.server.Environment;
+import org.apache.sshd.server.ExitCallback;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/**
+ * Unit tests for {@link MkCommand}.
+ * @author Mihai Andronache (amihaiemil@gmail.com)
+ * @version $Id$
+ * @since 1.6
+ */
+public final class MkCommandTest {
+    /**
+     * Check if {@link MkCommand} can start.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void mkCommandStarts() throws Exception {
+        final ByteArrayOutputStream output = new ByteArrayOutputStream();
+        final String commandText = "test mkcommand";
+        final MkCommand command = new MkCommand(commandText);
+        command.setOutputStream(output);
+        command.setExitCallback(Mockito.mock(ExitCallback.class));
```

Comment:
@mkordas Yes there are multiple possibilities, but I believe this is the best choice. Alternatives that I thought of are:  
1) We can declare anonymous classes for both ExitCallback and Environment interfaces. See gist: https://gist.github.com/amihaiemil/8c8d140b8e6e7db05ac2

2) We can declare anonymous class only for ExitCallback and set Environment to ``null``, but  that violates our no-nulls principle, doesn't it? See gist: https://gist.github.com/amihaiemil/040c2c515048f2e2399a

3) We would have our own mock implementations of Environment and ExitCallback - again, I think it's pointless to have them only for this simple test. No methods implementation matters for the test anyway. 

What do you think?